__tests__/util/codemirror-utils.test.js

```diff
@@ -0,0 +1,82 @@
+import * as codemirrorUtils from '../../src/util/codemirror-utils';
+
+const lineText = "Wubba lubba dub dub!";
+
+const mockPosFromIndex = [
+  { line: 0, ch: 0, },
+  { line: 0, ch: 20, },
+];
+
+const posFromIndex = jest.fn()
```

Comment:
Very slick, not immediately obvious whats being done. Put a explanatory comment here.